qiskit_optimization/problems/quadratic_program.py

```diff
@@ -869,6 +871,41 @@ def maximize(
             self, constant, linear, quadratic, QuadraticObjective.Sense.MAXIMIZE
         )
 
+    @staticmethod
+    def load(model: Any) -> "QuadraticProgram":
+        """Loads this quadratic program from an optimization model.
+
+        Note that this supports only basic functions of docplex as follows:
+        - quadratic objective function
+        - linear / quadratic constraints
+        - binary / integer / continuous variables
+
+        Args:
+            model: The optimization model to be loaded.
+
+        Returns:
+            the quadratic program corresponding to the model.
+
+        Raises:
+            QiskitOptimizationError: if the model contains unsupported elements.
+        """
+        return _load_model(model)
+
+    def save(self, translator: Optional[ModelTranslator] = None) -> Any:
```

Comment:
should we rename this to `export` and then maybe just incorporate the `export_to_lp_string` as well via `translator='lp'` or something like that?

Reply:
Thank you for your idea. `export_to_lp_string` came from docplex method. If we make docplex translator, it would be simpler to make `lp` as a translator.

I'm wondering the pair of names "load/save", "?/export", or "from_qp/to_qp". Word "import" and "from" are not allowed because they are reserved by Python syntax.

Reply:
I notice that we can deprecate other methods as well https://github.com/Qiskit/qiskit-optimization/pull/122#issuecomment-849341955

Reply:
I can add a `LPStringTranslator` that supports only `from_qp(qp) -> str` and does not support `to_qp(model) -> QP`. What do you think?

Reply:
Do you prefer naming `export` and `load`?

Reply:
I implemented your idea. Could you verify it?
https://github.com/Qiskit/qiskit-optimization/pull/122#discussion_r640105391